docs/creating-manifest/selecting-containers.md

```diff
@@ -275,7 +275,7 @@ The supported software stacks are categorized in the table below with specified
 
 |Stacks|Java|PHP|Ruby|Python|Nodejs|ii8|
 |--------|----------|-----------|-----------|--------|---------|---------|
-|engine|java6<br>java7<br>java8<br>jdk-9<br>jdk-10<br>jdk-1.8.0_144<br>jdk-1.8.0_152|php5.3<br>php5.4<br>php5.5<br>php5.6<br>php7<br>php7.1.13<br>php7.1.7<br>php7.2.1|ruby1.9<br>ruby2.0<br>ruby2.1<br>ruby2.2<br>ruby2.3<br>ruby2.4.1|python2.7<br>python3.3<br>python3.4<br>python3.5|nodejs6.11.5<br>nodejs6.12.3<br>nodejs8.9.0<br>nodejs8.9.4<br>nodejs9.0.0<br>nodejs9.4.0|.NET 4|
+|engine|jdk7<br>jdk8<br>jdk11<br>openj98<br>openj911<br>openj916<br>openjdk7<br>openjdk8<br>openjdk11<br>openjdk14<br>openjdk15<br>openjdk16<br>openjdk17<br>adoptopenjdk8<br>adoptopenjdk11<br>adoptopenjdk16<br>correttojdk8<br>correttojdk11<br>correttojdk15<br>correttojdk16<br>libericajdk8<br>libericajdk11<br>libericajdk14<br>libericajdk15<br>libericajdk16<br>zulujdk7<br>zulujdk8<br>zulujdk11<br>zulujdk13<br>zulujdk14<br>zulujdk15<br>zulujdk16<br>|php7.3<br>php7.4<br>php8.0|ruby2.5<br>ruby2.6<br>ruby2.7<br>ruby3.0|python3.6<br>python3.7<br>python3.8<br>python3.9|nodejs10.24.1<br>nodejs12.22.1<br>nodejs14.17.0<br>nodejs15.14.0<br>nodejs16.2.0|.NET 5|
```

Comment:
jdk?

Reply:
The list is up-to-date now https://github.com/vlobzakov/docs/pull/141/commits/c6b463f68d611c17544daf47f66f2bd7d60677e5